client/manager.go

```diff
@@ -43,5 +43,7 @@ type Storage interface {
 
 	GetClients(ctx context.Context, limit, offset int) (map[string]Client, error)
 
+	GetClientCount(ctx context.Context) (int, error)
```

Comment:
For a more homogenous naming, how about `CountClients()`, which is closer to `CountSubjectsGrantedConsentRequests`)